pkg/storage/storage_benchmark_test.go

```diff
@@ -348,10 +350,11 @@ func (g *generator) RandomString(length int) string {
 }
 
 func (g *generator) RandomMetricsPoint() MetricsPoint {
+	randomfactor += 1e10
```

Comment:
Problem we are trying to fix is that each sample of `CumulativeCpuUsed` metric (per container) should always be increasing. Could we make the code represent this behavior better?

Instead of creating some cumulative `randomfactor` and than adding some random value, could we just store last `CumulativeCpuUsed` we returned?

Reply:
done